core/src/main/java/com/ibm/wala/ipa/cha/ClassHierarchy.java

```diff
@@ -706,6 +708,97 @@ private void recursiveStringify(Node n, StringBuilder buffer) {
     }
   }
 
+  // Puts ClassHierarchy into a JSON readable variable that maps the class to a list of its
+  // subclasses
+  public Object toJson() {
+    // lines 713-715: initialize variables (subclass and dag) to store important values like hashmap
+    // of each iteration and a list of all maps of class to subclasses
+    HashMap<Object, Object> subclass = new HashMap<>();
+    Iterator<Node> children = root.getChildren();
+    ArrayList<HashMap<Object, Object>> dag = new ArrayList<>();
```

Comment:
I don't understand the purpose of this list.  As far as I can tell, every element of the list will be the exact same map.  What am I missing?